integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDomainInImage.java

```diff
@@ -223,6 +233,52 @@ void testIstioDomainHomeInImage() {
     logger.info("Application Access URL {0}", url);
     boolean checkApp = checkAppUsingHostHeader(url, domainNamespace + ".org");
     assertTrue(checkApp, "Failed to access WebLogic application");
+
+  }
+
+  /**
+   * Verify that Security Warning Tool does not detect any security warning
+   * messages on console.
+   * Ref https://jira.oraclecorp.com/jira/browse/OWLS-92319
+   * However due to WDT issue, Security warning (related to minimum umask) 
+   * are reported in Console.
+  */
+  
+  @Test
+  @Order(2)
+  @Disabled("Disable test based on issues found reported OWLS-92319")
+  @DisplayName("Verify the Security Warning in domain in image")
+  void testVerifySecurityWarnings() {
+
+    int istioIngressPort = getIstioHttpIngressPort();
+
+    logger.info("Istio Ingress Port is {0}", istioIngressPort);
+    if (isWebLogicPsuPatchApplied()) {
+      String curlCmd2 = "curl -j -sk --show-error --noproxy '*' "
+          + " -H 'Host: " + domainNamespace + ".org'"
+          + " --user " + ADMIN_USERNAME_DEFAULT + ":" + ADMIN_PASSWORD_DEFAULT
+          + " --url http://" + K8S_NODEPORT_HOST + ":" + istioIngressPort
+          + "/management/weblogic/latest/domainRuntime/domainSecurityRuntime?" 
+          + "link=none";
+
+      ExecResult result = null;
```

Comment:
Minor - I think this is redundant. Consider doing the initialization and command execution in the same line (like below).  

```
ExecResult result = assertDoesNotThrow( () -> exec(curlCmd2, true));
```